scripts/100-ping-check

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ping_checker() {
+	HOST="$1"
+	HOST_PING="$(ping -c 2 $HOST 2>&1 | grep '% packet' | awk -F'[%]' '{print $1}' | awk -F'[ ]' '{print $NF}')"
```

Comment:
when the host is not available, it takes this over 5 seconds to run, which is a little inconvenient. Any way to reduce this time?

Reply:
i added a timeout of two seconds is that okay?

Reply:
:+1: